### PR TITLE
Added tests for GetAllColumns

### PR DESCRIPTION
**Pull request by icnocop**

### CODE REVIEW

File: src/GUI/SSMSToolbox/SSMSToolbox.csproj [hunk 1]
```diff
@@ -560,7 +560,7 @@
       <HintPath>..\lib\v140\SQLEditors.dll</HintPath>
     </Reference>
     <Reference Include="SQLiteScripting">
-      <HintPath>..\lib\SQLiteScripting.dll</HintPath>
+      <HintPath>..\..\API\SQLiteScripting\bin\$(Configuration)\SQLiteScripting.dll</HintPath>
```
Comment (ErikEJ) [inline src/GUI/SSMSToolbox/SSMSToolbox.csproj]: No, the files are built from the API project, and copied to the lib folder

File: src/GUI/SqlCe40ToolboxExe/SqlCe40ToolboxExe.csproj [hunk 2]
```diff
@@ -716,7 +712,7 @@
     </PublishFile>
   </ItemGroup>
   <ItemGroup>
-    <EmbeddedResource Include="..\lib\SqlCeToolbox.DDEX4.dll">
+    <EmbeddedResource Include="..\SqlCeToolbox.DDEX4\bin\x86\$(Configuration)\SqlCeToolbox.DDEX4.dll">
```
Comment (ErikEJ) [inline src/GUI/SqlCe40ToolboxExe/SqlCe40ToolboxExe.csproj]: Keep using the lib folder. pls

File: src/GUI/SSMSToolbox/SSMSToolbox.csproj (hunk 1 above)
Comment (icnocop) [reply src/GUI/SSMSToolbox/SSMSToolbox.csproj]: I notice that you do this manually; is there a reason why it's not automatically done by the build?

I see `copytogui.cmd` but it doesn't seem like it's in the right place.

Comment (ErikEJ) [reply src/GUI/SSMSToolbox/SSMSToolbox.csproj]: Sorry, the project is not really properly setup for OSS :-(

**Comment by icnocop** [comment]
I reverted the changes I made so the dll references are picked up from .\lib as before.

Thank you.

**Comment by ErikEJ** [comment]
Great, thanks for your PRs!

**Pull request merged**